packages/design-system/src/scalars/components/toggle/toggle.test.tsx

```diff
@@ -1,75 +1,97 @@
 import React from "react";
-import { render, screen, fireEvent } from "@testing-library/react";
-import { describe, it, vi } from "vitest";
-import { ToggleField } from "./toggle-field";
+import { render, fireEvent, screen } from "@testing-library/react";
+import { Toggle } from "./toggle";
+import { vi } from "vitest";
 
 describe("Toggle Component", () => {
-  const mockOnChange = vi.fn();
+  const defaultProps = {
+    onChange: vi.fn(),
+    disabled: false,
+    checked: false,
+    required: false,
+  };
+
   beforeEach(() => {
     vi.clearAllMocks();
   });
```

Comment:
same that above, this shouldn't be needed here

Reply:
you are right, done